hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/impl/SqlClientPool.java

```diff
@@ -41,7 +41,21 @@
  */
 public class SqlClientPool implements ReactiveConnectionPool, ServiceRegistryAwareService, Configurable, Stoppable, Startable {
 
+	/**
+	 * @see SqlConnectOptions#setPreparedStatementCacheSqlLimit(int)
+	 */
+	public static final String PREPARED_STATEMENT_CACHE_SQL_LIMIT = "hibernate.vertx.prepared_statement_cache.sql_limit";
+	/**
+	 * @see SqlConnectOptions#setPreparedStatementCacheMaxSize(int)
+	 */
+	public static final String PREPARED_STATEMENT_CACHE_MAX_SIZE = "hibernate.vertx.prepared_statement_cache.max_size";
+	/**
+	 * @see PoolOptions#setMaxWaitQueueSize(int)
+	 */
+	public static final String MAX_WAIT_QUEUE_SIZE = "hibernate.vertx.pool.max_wait_queue_size";
```

Comment:
Do these options belong somewhere else if we expect a user to be setting them? We used to have an `AvailableReactiveSettings` interface for reactive-only settings, seems like that would be a proper place for these

Reply:
*Eventually*, yes. Indeed I *just added* a package called `common` which may be a reasonable place to put this.

But what I definitely don't want is classes with useful user-visible things just popping up randomly in packages that we haven't defined as part of the public API.